src/Symfony/Bundle/SecurityBundle/DependencyInjection/MainConfiguration.php

```diff
@@ -90,7 +90,10 @@ public function getConfigTreeBuilder()
                     ->defaultValue(SessionAuthenticationStrategy::MIGRATE)
                 ->end()
                 ->booleanNode('hide_user_not_found')->defaultTrue()->end()
-                ->booleanNode('always_authenticate_before_granting')->defaultFalse()->end()
+                ->booleanNode('always_authenticate_before_granting')
+                    ->defaultFalse()
+                    ->setDeprecated('symfony/security-bundle', '5.4', 'The "%node%" at path "%path%" is deprecated and will always be "false" in 6.0.')
```

Comment:
Can't it be removed in 6.0?

Reply:
Thanks for spotting this confusing wording. Yes, it'll be removed in 6.0